src/components/views/rooms/ReplyPreview.tsx

```diff
@@ -44,19 +46,17 @@ export default class ReplyPreview extends React.Component<IProps> {
         if (!this.props.replyToEvent) return null;
 
         return <div className="mx_ReplyPreview">
-            <div className="mx_ReplyPreview_section">
-                <div className="mx_ReplyPreview_header">
-                    <span>{ _t('Replying') }</span>
-                    <AccessibleButton
-                        className="mx_ReplyPreview_header_cancel"
-                        onClick={() => cancelQuoting(this.context.timelineRenderingType)}
-                    />
-                </div>
-                <ReplyTile
-                    mxEvent={this.props.replyToEvent}
-                    permalinkCreator={this.props.permalinkCreator}
-                />
+            <div className="mx_ReplyPreview_header">
+                <ReplyIcon />
+                { _t('Reply to') } &nbsp;
```

Comment:
This needs to include the sender profile as a `<tag/>` as the order of the sentence will vary per-language